iXBRLViewerPlugin/viewer/src/i18n/da/currencies.json

```diff
@@ -0,0 +1,13 @@
+{
+  "unitFormatGBP": "\u00A3",
+  "unitFormatUSD": "US \u0024",
+  "unitFormatEUR": "\u20AC",
+  "unitFormatMXN": "MXN \u0024",
+  "unitFormatUAH": "\u20B4",
+  "centsUSD": "cents",
+  "centsEUR": "cents",
+  "centsAUD": "cents",
+  "centsZAR": "cents",
+  "centsMXN": "cents",
```

Comment:
Out of curiosity, if these are all untranslated / the same as English, do they still need to be specified?